src/losses/functions.jl

```diff
@@ -425,7 +425,47 @@ function tversky_loss(ŷ, y; β=ofeltype(ŷ, 0.7))
     1 - num / den
 end
 
+"""
+    binary_focal_loss(ŷ, y; agg=mean, γ=ofeltype(ŷ, 2.0), ϵ=epseltype(ŷ))
```

Comment:
```suggestion
    binary_focal_loss(ŷ, y; agg=mean, γ=2, ϵ=eps(ŷ))
```
Even if not the exact signature, is informative enough

Reply:
Correct me if I'm wrong(and this is also a doubt). `γ=2` will get promoted to floating point when multiplied later. Is it more numerical efficient to declare `γ=2` than `γ=ofeltype(ŷ, 2.0)`?

Reply:
Does promotion bear an overhead cost?

Reply:
No I think @CarloLucibello wants you to leave out the `ofeltype` just in the docstring. The code can remain the same.

Reply:
Okay! My bad :)

Reply:
Yes I meant just the docstring, but now that think about it `γ=2` is actually a better default since integer exponentiation takes a different and faster path